src/main/java/prefs/ConfigFile.java

```diff
@@ -0,0 +1,131 @@
+package prefs;
+
+import com.google.gson.*;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import util.HTLog;
+
+import java.io.*;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.Optional;
+
+/**
+ * A general class for HubTurbo config files.
```

Comment:
~~`A general class for HubTurbo config files.`~~ `Represents a HubTurbo Config file`